server/npg/porch/endpoints/pipelines.py

```diff
@@ -43,29 +44,40 @@ async def get_pipelines(db_accessor=Depends(get_DbAccessor)) -> List[Pipeline]:
 
```

Comment:
A comment for the endpoint for getting all pipelines. The summary states that uri can be used to filter, but it is not passed.